lib/process-services-cloud/src/lib/task/task-header/components/task-header-cloud.component.ts

```diff
@@ -130,8 +132,12 @@ export class TaskHeaderCloudComponent implements OnInit, OnDestroy, OnChanges {
                 } else {
                     this.refreshData();
                 }
+                this.isLoading = false;
```

Comment:
can't we have 
 .pipe(finalize(() => (this.isLoading = false)))

Reply:
Updated. Thanks